modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -293,6 +297,14 @@ def submit_evidence_to_central_mail!
       evidence_submissions&.each(&:submit_to_central_mail!)
     end
 
+    def email_identifier
+      return { id_type: 'email', id_value: email } if email.present?
+
+      icn = mpi_veteran.mpi_icn
+
+      icn.present? ? { id_type: 'ICN', id_value: icn } : {}
```

Comment:
I realized that if, for some reason, this is called on a record with no email or ICN (very unlikely), this would return `nil`, which could raise an error if accessed with the assumption that the data were available, so I added a fallback to an empty hash.

(As in my original PR for this ticket, these `email_identifier` methods are moved to the public sections of the models so that we can call and use them in the `AppealSubmittedJob` when building VANotify arguments)